### PR TITLE
DB2 Error Handling

### PR DESCRIPTION
**Pull request by nlraley**
Fixes for error handling.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@morgul/knex-db2",
-  "version": "0.4.0",
```
Comment (Morgul) [inline package.json]: Don't bump versions in pull requests like this. It's up to me to decide to release a new version.

File: src/index.ts [hunk 2]
```diff
@@ -234,7 +234,9 @@ class DB2Client extends knex.Client
             }
             catch (err : any)
             {
-                this.printError(err);
+                this.printError(err);                
+                console.error(err);
```
Comment (Morgul) [inline src/index.ts]: Don't add a `console.log`, that's the point of `printError`.

File: package.json (hunk 1 above)
Comment (nlraley) [reply package.json]: Should be fixed in latest commits.

File: src/index.ts (hunk 2 above)
Comment (nlraley) [reply src/index.ts]: Should be fixed in latest commits.

**Pull request merged**